composer.json

```diff
@@ -20,56 +20,63 @@
         }
     ],
     "require": {
-        "php": ">=8.0",
-        "composer/installers": "^1.7",
+        "php": ">=8.2",
+        "composer/installers": "^1.12",
         "cweagans/composer-patches": "^1.7",
         "drupal/admin_denied": "^2.0",
         "drupal/allowed_formats": "^2.0",
-        "drupal/components": "^2.4",
-        "drupal/config_filter": "^2.2",
-        "drupal/config_split": "^2.0.0-beta4",
-        "drupal/core-composer-scaffold": "^9.3",
-        "drupal/core-dev": "^9.3",
-        "drupal/core-project-message": "^9.3",
-        "drupal/core-recommended": "^9.5",
-        "drupal/csp": "^1.16",
+        "drupal/ckeditor": "^1.0",
```

Comment:
We should be moving to CKEditor5, which is in core, but that can happen in another ticket.
Edit - just seen we do already have CKEditor5 enabled - do we need this contrib module?

Reply:
Yes, this needed for the upgrade. The prod ODSG site is still using CKeditor 4. https://github.com/UN-OCHA/odsg8-site/pull/231 was not deployed. After the upgrade this module and others like `seven` can be removed but they need to be present so that Drupal can uninstall them.